logging/google/cloud/logging/client.py

```diff
@@ -76,6 +76,11 @@ class Client(JSONClient):
     _connection_class = Connection
     _logging_api = _sinks_api = _metrics_api = None
 
+    def __init__(self, project=None, credentials=None,
+                 http=None, use_gax=True):
+        super(Client, self).__init__(project, credentials, http)
+        self.use_gax = use_gax
```

Comment:
I think I'd prefer using `_USE_GAX` here. Instead let the default be `use_gax=None`, then

```python
if use_gax is None:
     self.use_gax = _USE_GAX
else:
     self.use_gax = use_gax
```

Also, any reason for `self.use_gax` to be public?